src/Npgsql.Json.NET/NpgsqlJsonNetExtensions.cs

```diff
@@ -40,22 +40,28 @@ public static class NpgsqlJsonNetExtensions
         /// <param name="mapper">The type mapper to set up (global or connection-specific)</param>
         /// <param name="jsonbClrTypes">A list of CLR types to map to PostgreSQL jsonb (no need to specify NpgsqlDbType.Jsonb)</param>
         /// <param name="jsonClrTypes">A list of CLR types to map to PostgreSQL json (no need to specify NpgsqlDbType.Json)</param>
-        public static INpgsqlTypeMapper UseJsonNet(this INpgsqlTypeMapper mapper, Type[] jsonbClrTypes = null, Type[] jsonClrTypes = null)
+        /// <param name="jsonSerializerSettings">Optional settings to customize JSON serialization</param>
+        public static INpgsqlTypeMapper UseJsonNet(
+            this INpgsqlTypeMapper mapper, 
+            Type[] jsonbClrTypes = null, 
+            Type[] jsonClrTypes = null,
+            Newtonsoft.Json.JsonSerializerSettings jsonSerializerSettings = null
+        )
```

Comment:
Add a namespace using and remove the full name. The last parameter name can be simplified to `settings`.

Also that change breaks binary compatibility since default parameter values are emitted by the compiler in call sites, but it's a preview package. Therefore, that change would be okay probably.